ehitchhiking-ui/src/app/shared/common/styles/colour-constants.scss

```diff
@@ -0,0 +1,147 @@
+$light-blue: #172651;
+$blue: #0a1339;
+$dark-blue: #0a122a;
+
+$green: #698f3f;
+$light-green: #84b149;
+$dark-green: #55773a;
+
+$beige: #e7decd;
+$brown: #804e49;
+$dark-brown: #4b2f24;
+
+$light-red: #FB5B74;
+$red: #F20026;
+$dark-red: #B5001D;
+
+$custom-primary-colour: (
+  50 : $light-green,
+  100 :$green,
+  200 :$dark-green,
+  300 : $green,
+  400 : $green,
+  500 : $green,
+  600 : $green,
+  700 : $dark-green,
+  800 : $dark-green,
+  900 : $dark-green,
+  A100 : $beige,
+  A200 : $brown,
+  A400 : $brown,
+  A700 : $dark-brown,
+  contrast: (
+    50 : #000000,
+    100 : #000000,
+    200 : #000000,
+    300 : #000000,
+    400 : #000000,
+    500 : #ffffff,
+    600 : #ffffff,
+    700 : #ffffff,
+    800 : #ffffff,
+    900 : #ffffff,
+    A100 : #000000,
+    A200 : #000000,
+    A400 : #000000,
+    A700 : #000000,
+  )
+);
+
+$custom-accent-colour: (
```

Comment:
pls rename to ehitchhiking-accent-color-setting

Reply:
done